vars/kola.groovy

```diff
@@ -140,7 +147,7 @@ def call(params = [:]) {
             // normal run (without reprovision tests because those require a lot of memory)
             id = marker == "" ? "kola" : "kola-${marker}"
             ids += id
-            runKola(id, 'run', "--tag='!reprovision' --parallel=${parallel} ${args}")
+            runKola(id, 'run', "--tag='!reprovision' ${args} --parallel=${parallel}")
```

Comment:
I moved the position of of `${args}` here so the output command would be cleaner. Instead of 
```text
kola run --tag=!reprovision --parallel=5 --tag=!openshift 
```
it'll look like:
```text
kola run --tag=!reprovision --tag=!openshift --parallel=5 
```

Reply:
Sounds good, though for consistency, let's also do that on line 124?

Reply:
Done!